### ISSUE TITLE
After installing your addon, ember server-start fails

### ISSUE THREAD

**Issue by wuarmin** [smith-carson/ember-simplemde#2]
Hello, 
I want to try out your addon, but had no success. After 
```ember install ember-simplemde```
which executes without error, ember-server start fails with:
```
Cannot find module '/home/dev/content-administration/bower_components/ember/ember-template-compiler.js'
Error: Cannot find module '/home/dev/content-administration/bower_components/ember/ember-template-compiler.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Class.htmlbarsOptions (/home/dev/content-administration/node_modules/ember-simplemde/node_modules/ember-cli-htmlbars/ember-addon-main.js:99:25)
    at Object.toTree (/home/dev/content-administration/node_modules/ember-simplemde/node_modules/ember-cli-htmlbars/ember-addon-main.js:31:43)
    at /home/dev/content-administration/node_modules/ember-cli-preprocess-registry/preprocessors.js:180:26
    at Array.forEach (native)
    at processPlugins (/home/dev/content-administration/node_modules/ember-cli-preprocess-registry/preprocessors.js:178:11)
    at module.exports.preprocessTemplates (/home/dev/content-administration/node_modules/ember-cli-preprocess-registry/preprocessors.js:161:10)
```
So far I could not figure out, how to solve the problem. 
my env:
```
ember-cli: 2.12.1
node: 6.9.4
os: linux x64

```

**Comment by marceloandrader** [comment]
hey @wuarmin yeah looks like it's a compatibility issue with ember 2.12, now ember is not installed as bower dependency, instead using npm as ember-source, I think we need to upgrade ember-cli-html-bars which is the one requesting that file not found.

I hope to do it during the day.

**Comment by wuarmin** [comment]
Thank you

**Comment by marceloandrader** [comment]
@wuarmin   I've published version 0.3.0 that fixes the issue.

**Comment by wuarmin** [comment]
Thanks :+1:
